tests/Validation/ValidationValidatorTest.php

```diff
@@ -4846,12 +4846,6 @@ public function testValidateImage()
         $v = new Validator($trans, ['x' => $file2], ['x' => 'image']);
         $this->assertTrue($v->passes());
 
-        $file2 = $this->getMockBuilder(UploadedFile::class)->onlyMethods(['guessExtension', 'getClientOriginalExtension'])->setConstructorArgs($uploadedFile)->getMock();
-        $file2->expects($this->any())->method('guessExtension')->willReturn('jpg');
-        $file2->expects($this->any())->method('getClientOriginalExtension')->willReturn('jpg');
-        $v = new Validator($trans, ['x' => $file2], ['x' => 'image']);
-        $this->assertTrue($v->passes());
-
```

Comment:
Does see why this need to be removed.

Reply:
This assertion is duplicated with the above - lines 4843-4847

Reply:
Might be best to not remove this on a PR that add additional feature. We can always submit test improvements separately.

Reply:
ok, will fix this in a minute and add another PR!

Reply:
@crynobone done. Let me know if now this is ok!